charts/humio-operator/templates/operator-deployment.yaml

```diff
@@ -64,6 +64,14 @@ spec:
           value: "humio-operator"
         - name: USE_CERTMANAGER
           value: {{ .Values.certmanager | quote }}
+        {{- if .Values.image }}
+        - name: HUMIO_OPERATOR_DEFAULT_IMAGE
+          value: {{ .Values.image }}
+        {{- end }}
```

Comment:
Given that the golang source code already checks if the value is `!= ""` maybe we could simplify this and just do:
```suggestion
        - name: HUMIO_OPERATOR_DEFAULT_IMAGE
          value: {{ .Values.defaultImage | quote }}
```
that makes it read much more in line with the one above. In this case I prefixed the `image` with `default` to make it a bit more explicit as to what it does. Perhaps it should even be `defaultHumioCoreImage`? Simply calling it `image` feels a bit too generic to me and doesn't really say much about what it is when you see it.

One could say the same about the env var `HUMIO_OPERATOR_DEFAULT_IMAGE` as that doesn't say much about what image we're talking about, so perhaps that should be more explicit `HUMIO_OPERATOR_DEFAULT_HUMIO_CORE_IMAGE`? Users are not expected to explicitly set that env var anyway, but rather install the operator with helm, so doesn't matter if the env var name is a little long.

And then we'd probably also need to add an entry in values.yaml where it sits next to `certmanager: ...` with perhaps `defaultImage: ""`. That way the values.yaml does have a mention of this setting being there, whereas this way users wouldn't know unless they go digging through the template. Perhaps we could also include comments in the values.yaml next to the config on what the field does.